### PR TITLE
Fix read fan_fault=1 and send log 

### PR DESCRIPTION
**Pull request by jostar-yang**
Fix read fan_fault=1 and send log when fan insert sometimes
**- What I did**
Fix read fan_fault=1 and send log when fan insert sometimes
**- How I did it**
Add check fan_fault  when fan_status=fan_fault . If fan_fault become to zero, log "FAN-N becomes operational"
**- How to verify it**
Run monitor_fan.py. and check log as below,
 root        : INFO     FAN-6 present is detected
root        : WARNING  Alarm for FAN-6 failed is detected
root        : INFO     FAN-6 becomes operational
**- Description for the change log**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 # HISTORY:
 #    mm/dd/yyyy (A.D.)
 #    5/15/2019:  Jostar create for as4630-54pe
+#    9/25/2019:  Jostar fix that get fan_faul=1 when fan_insert and read cpld
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py]: Typo: "fan_faul" -> "fan_fault"

File: platform/broadcom/sonic-platform-modules-accton/as5835-54t/utils/accton_as5835_54t_monitor_fan.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 # HISTORY:
 #    mm/dd/yyyy (A.D.)
 #    5/27/2019:  Brandon_Chuang create
+#    9/25/2019:  Jostar fix that get fan_faul=1 when fan_insert and read cpld
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as5835-54t/utils/accton_as5835_54t_monitor_fan.py]: Typo: "fan_faul" -> "fan_fault"

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/accton_as7726_32x_monitor_fan.py [hunk 3]
```diff
@@ -18,7 +18,8 @@
 # ------------------------------------------------------------------
 # HISTORY:
 #    mm/dd/yyyy (A.D.)
-#    7/2/2018:  Jostar create for as7326-56x
+#    7/2/2018:  Jostar create for as7726-32x
+#    9/25/2019:   Jostar fix that get fan_faul=1 when fan_insert and read cpld
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/accton_as7726_32x_monitor_fan.py]: Typo: "fan_faul" -> "fan_fault"

File: platform/broadcom/sonic-platform-modules-accton/as9716-32d/utils/accton_as9716_32d_monitor_fan.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
 # HISTORY:
 #    mm/dd/yyyy (A.D.)
 #    12/13/2018:  Jostar create for as9716-32d
+#    9/25/2019:   Jostar fix that get fan_faul=1 when fan_insert and read cpld
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as9716-32d/utils/accton_as9716_32d_monitor_fan.py]: Typo: "fan_faul" -> "fan_fault"

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py (hunk 1 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py]: Fixed to "fan_fault".

File: platform/broadcom/sonic-platform-modules-accton/as5835-54t/utils/accton_as5835_54t_monitor_fan.py (hunk 2 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as5835-54t/utils/accton_as5835_54t_monitor_fan.py]: Fixed to "fan_fault".

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/accton_as7726_32x_monitor_fan.py (hunk 3 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as7726-32x/utils/accton_as7726_32x_monitor_fan.py]: Fixed to "fan_fault".

File: platform/broadcom/sonic-platform-modules-accton/as9716-32d/utils/accton_as9716_32d_monitor_fan.py (hunk 4 above)
Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as9716-32d/utils/accton_as9716_32d_monitor_fan.py]: Fixed to "fan_fault".

File: platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py [hunk 5]
```diff
@@ -185,7 +185,7 @@ def main(argv):
     monitor = device_monitor(log_file, log_level)
     while True:
         monitor.manage_fan()
-        time.sleep(3)
+        time.sleep(3)
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py]: Remove trailing spaces

Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as4630-54pe/utils/accton_as4630_54pe_monitor_fan.py]: Done to remove not needed spaces

**Comment by jostar-yang** [comment]
When will you can merge this PR?  Is there problem for this PR?

**Comment by jleveque** [comment]
Retest vs please

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jostar-yang** [comment]
When will you merge this PR? Our customer git clone master code, but found this PR still not be merged. This PR seems take a long time.

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
Retest broadcom please

**Comment by jleveque** [comment]
Retest mellanox please

**Comment by jostar-yang** [comment]
Retest vs please

**Comment by jostar-yang** [comment]
Close this PR due to there are too mant commit in this PR. I will re-send new PR.

**Pull request closed without merging**